service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -54,14 +60,30 @@ class DatasetServiceTest {
   private final AuthenticatedUserRequest user = mock(AuthenticatedUserRequest.class);
   private final DatasetId datasetId = new DatasetId(UUID.randomUUID());
   private final String sourceId = "sourceId";
+  private final String workspaceSourceId = "abc-def-workspace-id";
   private final String metadata = """
         {"name":"name"}""";
   private final Dataset dataset =
       new Dataset(datasetId, sourceId, StorageSystem.EXTERNAL, metadata, null);
+  private final Dataset tdrDataset =
```

Comment:
I moved these out because it seemed weird to just have an external dataset. Open to moving back, but it seemed like we were repeating things we didn't need to.